emgapianns/management/lib/sanity_check.py

```diff
@@ -186,7 +229,7 @@ def run_coverage_check_assembly(self):
         """
         logging.info("Running coverage check for assembly data.")
         taxa_folder = os.path.join(self.dir, "taxonomy-summary")
-        if not os.path.exists(taxa_folder):
+        if not os.path.exists(taxa_folder) and self.result_status not in ['no_tax', 'no_cds_tax']:
```

Comment:
I'm not sure but shouldn't this be an `or`? Feel free to ginore this comment if you are sure is an `and`